docs/Deploying/Configuration.md

```diff
@@ -2287,9 +2287,25 @@ It is strongly recommended that at least one admin user is set up to use the tra
 configuration problems.
 
 > Individual users must also be set up on the IDP before signing in to Fleet.
+
 ### Enabling SSO for existing users in Fleet
 As an admin, you can enable SSO for existing users in Fleet. To do this, go to the Settings page, then click on the Users tab. Locate the user you want to enable SSO for and on the Actions dropdown menu for that user, click on "Enable single sign-on."
 
+### Enabling JIT user provisioning
+
+Admins can enable JIT provisioning to automatically create an account the first time a user logs in with SSO. To do this, go to **Settings > Organization settings > SAML single sign-on options** and check "_Automatically create Observer user on Login_".
+
+Users created via JIT provisioning are assigned the Observer role, and their email and full name are extracted from the SSO response. For this to work correctly make sure that:
```

Comment:
Is there a yaml config for this?

Reply:
Ah I see we have that below. Can we document that as well as a way to turn it on?

Reply:
seems like we don't have any of the sso configs documented below, but they are in the example configs here https://fleetdm.com/docs/using-fleet/configuration-files

I will double check if they can be really configured like that and update accordingly

Reply:
I think we should put that in here going forward. It is not obvious to look in that example doc.

Reply:
I agree, I will double check these options can be configured via config files and add details here if so.

Reply:
@zhumo I have verified that SSO configs can only be changed via the API, I believe we should probably add an issue and let users configure this feature via yaml configs as well.

For now, I have removed `sso_settings` from the yaml examples.

Reply:
We found that applying SSO configs does not work with `fleetctl serve -c` but does work with `fleetctl apply`. I think it should work under both contexts. Issue created here: https://github.com/fleetdm/fleet/issues/7208

cc: @chiiph @noahtalerman

Reply:
The config in `fleet serve` is one that needs a server restart to be effective (eg: mysql server URL changes). While the config that gets applied with `fleetctl apply` is one that can change at runtime.

We can redefine these boundaries, but before we do so, this setting should only be changeable with `fleetctl apply`. Having two configs can be confusing. Merging them into one config system is not necessarily clearer either though.

The types of user that changes each is different as well. A devops person running fleet is in charge of the `fleet serve` one, while a fleet admin (which might not be a devops person at all) can change the other.

Reply:
> this setting should only be changeable with `fleetctl apply`

I agree with Tomas here. 

I think it's ok that JIT provisioning settings can only be changed by running `fleetctl apply -f config.yml` (using the `config` YAML document). 

As a devops person starting or restarting the Fleet server with the `fleet serve` command, I don't want to worry about setting the JIT provisioning settings.

Reply:
cc @zhumo

Reply:
Great! works for me. Gonna close the other one then.